dashboard/app/views/pd/application/teacher_application_mailer/admin_approval_completed_teacher_receipt.html.haml

```diff
@@ -4,7 +4,7 @@
 
 %p
   Your Administrator/School Leader has approved your application. Your application
-  is now considered complete and we will review it and get back to you in 2 to 4 weeks.
+  is now considered complete and we will review it and get back to you in 2-4 weeks.
```

Comment:
@dmcavoy Curious on if we need to make this change? I know it's a small nit but I remember last year someone leaving feedback to try and make these consistently use "to" rather than "-". I don't fully remember the reason so totally good either way, more of a consistency thing since I know elsewhere we made sure to use "to".

Reply:
In the spec, I think most emails were "2-4" instead of "2 to 4". I have no opinion, but I leaned towards what was in the spec

Reply:
I'm fine either way so lets just pick one and stay consistent. Whatever we use most we should use here.

Reply:
All instances are 2-4 so I'll leave it as-is